schema.json

```diff
@@ -936,15 +942,18 @@
               "conductivity",
               "current",
               "digital input",
+              "digital output",
```

Comment:
A digital output is not a sensor

If needed, we can add a property `actuators`